pr_testing/run-pr-comparisons

```diff
@@ -398,10 +398,21 @@ for WF in ${WORKFLOWS_TO_COMPARE//,/ }; do
   WF_DIR=`echo $WF | cut -d "/" -f1`
   WF_NUMBER=`echo $WF | cut -d'_' -f1`
   WF_FILE=$(basename $WF)
-  if [ -f $WORKSPACE/data/$COMPARISON_RELEASE/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt ] ; then
-    $CMS_BOT_DIR/comparisons/compare-maxmem.py $WORKSPACE/data/PR-${PR_NUM}/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt $WORKSPACE/data/$COMPARISON_RELEASE/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt > $OUTPUT_DIR/${WF_NUMBER}.txt 2> $OUTPUT_DIR/${WF_NUMBER}.err || true
+  if [ -f $WORKSPACE/data/$COMPARISON_RELEASE/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt  ] ; then
+    if [ -f $WORKSPACE/data/PR-${PR_NUM}/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt ];then
+        $CMS_BOT_DIR/comparisons/compare-maxmem.py $WORKSPACE/data/PR-${PR_NUM}/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt $WORKSPACE/data/$COMPARISON_RELEASE/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt > $OUTPUT_DIR/${WF_FILE}.json 2> $OUTPUT_DIR/${WF_FILE}.err || true
+    fi
+    if [ -f $WORKSPACE/data/PR-${PR_NUM}/$WF_DIR/maxmem_profile_${WF_FILE}.txt ] ; then
+        $CMS_BOT_DIR/comparisons/compare-maxmem.py $WORKSPACE/data/PR-${PR_NUM}/$WF_DIR/maxmem_profile_${WF_FILE}.txt $WORKSPACE/data/$COMPARISON_RELEASE/$WF_DIR/maxmem_profile_${WF_NUMBER}.txt > $OUTPUT_DIR/${WF_FILE}.json 2> $OUTPUT_DIR/${WF_FILE}.err || true
+    fi
```

Comment:
@gartung , I do not understand this block ? Why are you looking for `maxmem_profile_${WF_FILE}.txt` . both [baseline](https://cmssdt.cern.ch/SDT/jenkins-artifacts/ib-baseline-tests/CMSSW_14_2_X_2024-09-23-1100/el8_amd64_gcc12/-GenuineIntel/matrix-results/) and [PR relvals](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ffff10/41711/runTheMatrix-results/) should have `maxmem_profile_${WF_NUMBER}.txt`

Reply:
I needed to rename the files produced by run-ib-pr-matrix.sh so I could use that to get the workflow number and name. These are then used to make a link in the table to the directory with the log files.

Reply:
I suppose I could get that from the directory name that maxmem_profile*.txt is in.